lib/structuraid_core/engineering/analysis/footing/centric_combined_two_columns.rb

```diff
@@ -0,0 +1,95 @@
+require 'byebug'
+
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        class CentricCombinedTwoColumns < Base
+          include Utils::BasicGeometry
+          include Utils::Centroid
+          include Utils::OneWayShear
+          include Utils::OneWayMoment
+
+          def initialize(footing:, loads_from_columns:, section_direction:)
+            if ORTHOGONALITIES.none?(section_direction)
+              raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+            end
+
+            @footing = footing
+            @loads_from_columns = loads_from_columns
+            @section_direction = section_direction
+          end
+
+          def build_geometry
+            coordinates_system = footing.coordinates_system
+
+            coordinates_system.clear_locations
+            relativize_loads
+
+            aligner_vector = coordinates_system.last_location_vector
+            coordinates_system.align_axis_1_with(vector: aligner_vector)
+
+            include_edges_locations_to_coordinates_system
+          end
+
+          def reaction_at_first_column
+            -(solicitation * @footing.horizontal_area + reaction_at_second_column)
+          end
+
+          def reaction_at_second_column
+            long_1 = long_border_to_first_column
+            long_2 = long_first_column_to_second_column
+            long_3 = long_second_column_to_border
```

Comment:
I think the right term for these names is `length` instead of `long`

Reply:
ok, done!